packages/plugin-list/test/Checklist.test.ts

```diff
@@ -3081,14 +3261,7 @@ describePlugin(List, testEditor => {
             });
         });
     });
-    describe('toggleChecked', () => {
-        it('should throw if we try to check an outside item', async () => {
-            const checklist = new ListNode({ listType: ListType.CHECKLIST });
-            const node = new CharNode({ char: 'a' });
-            expect(() => {
-                checklist.check(node);
-            }).to.throw();
-        });
```

Comment:
🙄

Reply:
As far as stupid tests go... A relic from when `ListNode` had a list of its checked children... Then `check` was called on the list to check one of its children :open_mouth: So this test checked that calling `check` on a list to check a node that wasn't one of its children made the whole editor crash. Because that would be best :shrug: